cmd/kots/cli/config/dev-cfg.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Let's move this file to the root of the repo and call it `kots-cli-dev.json` or something that mentions kots CLI.

Reply:
will leave it in the cli dir as this is specific to cli. discussed with @divolgin